frontend/src/components/Tabcordian.tsx

```diff
@@ -39,19 +39,19 @@ const Tabcordian: FC<TabcordianProps> = ({
 
     return (
         <div className="flex flex-col md:my-16">
-            <div className="hidden md:flex md:mx-16 md:justify-between px-14 md:space-x-6 rounded-2xl shadow-[0_4px_12px_rgba(0,0,0,0.12)] md:z-20 bg-white">
+            <div className="hidden md:flex md:mx-4 lg:mx-16 md:justify-between md:px-6 lg:px-14 md:space-x-4 lg:space-x-6 rounded-2xl shadow-[0_4px_12px_rgba(0,0,0,0.12)] md:z-20 bg-white">
```

Comment:
I wouldn't use `md:` other then for `md:flex` because from by default it's hidden and only display on `md` this goes for the button classes as well.

Reply:
I will have to leave in some of the md where md and lg differ, but I will scrub the others.